### PR TITLE
Port to mageia

### PR DESCRIPTION
**Pull request by Drotlorm**
FIX some installation problem for GLPI, Backuppc, Mail monitoring
FIX pip upgrade, missing OpenSSH package

Mail monitoring, GLPI and LDAP install still don't work

Problem remaining
For GLPI :
cliinstall execution

For LDAP :
User database creation
User add to database

Mailmonitoring :
Missing package for mageia (XFS-defrag)

### CODE REVIEW

**Comment by neoclust** [comment]
Hi,

the PR needs to be done against port_to_mageia branch of this repository?

can you ask a new PR ?

File: install/install-pulse [hunk 1]
```diff
@@ -1023,17 +1023,22 @@ install_backuppc() {
 
 initialise_glpi_db() {
     if [ ! -f /usr/share/glpi/install/cliinstall.php ]; then
+    			#check if the directory exist or not for the pushd
+    			if [ ! -d /usr/share/glpi/install/ ]; then
+    				if [ ! -d /usr/share/glpi/ ]; then
+    					mkdir /usr/share/glpi/
+    				fi
+    				mkdir /usr/share/glpi/install/
```
Comment (neoclust) [inline install/install-pulse]: this part is useless as /usr/share/glpi/install/ is provided by the glpi rpm.

File: install/install-pulse [hunk 2]
```diff
@@ -1040,7 +1045,7 @@ initialise_glpi_db() {
     echo "CREATE user 'glpi'@'localhost';" | mysql -u root -p${ROOT_PASSWORD}
 
     sed -i 's!glpi-9.1.6-empty!glpi-9.1-empty!' /usr/share/glpi/install/cliinstall.php
-    php /usr/share/glpi/install/cliinstall.php --db=glpi --user='root' --pass=${ROOT_PASSWORD} --lang=fr_FR --force
+    ./usr/share/glpi/install/cliinstall.php --db=glpi --user='root' --pass=${ROOT_PASSWORD} --lang=fr_FR --force
```
Comment (neoclust) [inline install/install-pulse]: this line is not wrong, but i don't see the need of removing the php command.

Comment (neoclust) [inline install/install-pulse]: This line is not wrong but i don't see the need of the removal of this php command.

**Comment by neoclust** [comment]
closing. 

I merged your patches manually

File: install/install-pulse (hunk 1 above)
Comment (Drotlorm) [reply install/install-pulse]: It's true but you use the fonction *initialise_glpi_db* before the installation of glpi rpm.

But you need the directory in the fonction

File: install/install-pulse (hunk 2 above)
Comment (Drotlorm) [reply install/install-pulse]: It's better to use php, but it's currently not working.

I'm on it at the moment.

File: install/install-pulse (hunk 1 above)
Comment (neoclust) [reply install/install-pulse]: then it is misplaced :)

Comment (neoclust) [reply install/install-pulse]: see commit 7bc42d5e5bd96e753eacf112c11a5f9680634045  it should be better

Comment (Drotlorm) [reply install/install-pulse]: Yes it is !

**Pull request closed without merging**